### DISCUSSION TITLE
Feature Request: Action support in UI

### DISCUSSION THREAD

**Discussion by ncodee** [Clooos/Bubble-Card#307] [Feature requests]
Currently, the `button` and `slider` card don't support editing the`tap_action, hold_action, double_tap_action` through the UI. Would be great if you can add support for easy editing, rather instead of having to edit **[Actions](https://www.home-assistant.io/dashboards/actions/)** it through yaml, thanks.

**Comment by Clooos** [comment]
Hi and thanks for your feedback, I still have to add that indeed, this is just a bit more complex. 

This is why I'm postponing that for too long 😅
